src/call.cpp

```diff
@@ -941,6 +941,38 @@ bool call::connect_socket_if_needed()
 
         sipp_customize_socket(call_socket);
 
+        if (peripsocket) {
+            struct sockaddr_storage saddr;
+            char peripaddr[256];
+            char *tmp = peripaddr;
+            getFieldFromInputFile(ip_file, peripfield, NULL, tmp);
+            struct addrinfo * h ;
+            struct addrinfo   hints;
+            memset((char*)&hints, 0, sizeof(hints));
+            hints.ai_flags  = AI_PASSIVE;
+            hints.ai_family = PF_UNSPEC;
```

Comment:
This should be `AF_UNSPEC`. The distinction is kinda moot but we're talking about the address family, not the protocol family.